src/pages/AboutUs/AboutUs.jsx

```diff
@@ -0,0 +1,75 @@
+const AboutUs = () => {
+  return (
+    <>
+      <div className={styles.mainContainer}>
+        <div className={styles.textContainer}>
+          <h1>Why the Learning Hub</h1>
+          <p>
+            We created this app to revolutionize the learning experience by
+            making it engaging, interactive, and accessible to everyone. In
+            today's fast-paced world, traditional learning methods can fall
+            short in keeping students motivated and involved. Our app addresses
+            this by combining courses, quizzes, and by providing administrators
+            with robust tools to manage student progress and course content. We
+            aim to streamline the educational process and enhance the overall
+            learning experience. Our goal is to empower learners and educators
+            alike, making education more effective and enjoyable for all.
+          </p>
+        </div>
+        <div className={styles.creatorsContainer}>
+          <div className={styles.textContainer}>
+            <h1>Meet the team</h1>
+            <p>Developers</p>
+          </div>
+          <div className={styles.cardsContainer}>
+            <div className={styles.card}>
+              <img src={user} alt='User Icon' className={styles.userIcon} />
+              <h1>Edith</h1>
+              <p>Back End Developer</p>
+            </div>
+            <div className={styles.card}>
+              <img src={user} alt='User Icon' className={styles.userIcon} />
+              <h1>Sonali</h1>
+              <p> Back End Developer</p>
+            </div>
+            <div className={styles.card}>
+              <img src={user} alt='User Icon' className={styles.userIcon} />
+              <h1>Olga</h1>
+              <p>Front End Developer</p>
+            </div>
+            <div className={styles.card}>
+              <img src={user} alt='User Icon' className={styles.userIcon} />
+              <h1>Laura</h1>
+              <p>Front End Developer</p>
+            </div>
+            <div className={styles.card}>
+              <img src={user} alt='User Icon' className={styles.userIcon} />
+              <h1>John</h1>
+              <p>Front End Developer</p>
+            </div>
+          </div>
+        </div>
+        <div className={styles.creatorsContainer}>
+          <div className={styles.textContainer}>
+            <p>Mentors</p>
+          </div>
+          <div className={styles.cardsContainer}>
+            <div className={styles.card}>
+              <img src={user} alt='User Icon' className={styles.userIcon} />
+              <h1>Eli</h1>
+              <p>Software Developer</p>
+            </div>
+            <div className={styles.card}>
+              <img src={user} alt='User Icon' className={styles.userIcon} />
+              <h1>Mattew</h1>
+              <p>Software Developer</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </>
+  );
+};
+export default AboutUs;
```

Comment:
Just a suggestion: you can use our real images in the <img> cards if the team agrees.

Reply:
For the sake of time, I wanted to have the about page done by this weekend, which is why i didn't I ask for picture from the team!